packages/create-project/src/create-project.ts

```diff
@@ -184,43 +187,16 @@ export async function createApp({
 
   if (await tryGitInit(root, gitInit)) {
     console.log('Initialized a git repository.');
-    console.log();
   }
 
-  let cdpath: string;
-  if (path.join(originalDirectory, appName) === appPath) {
-    cdpath = appName;
-  } else {
-    cdpath = appPath;
-  }
+  const messageFile = path.join(root, '.post-create');
```

Comment:
We should not fail the process if this file does not exist IMHO because this tool accepts arbitrary URLs.

Reply:
Oops! 😄